### DISCUSSION TITLE
How can I user runtimeConfig public env variables outside setup context (I just want to use a public env var in .ts module)

### DISCUSSION THREAD

**Discussion by Moataz-Abdulbaqi** [nuxt/nuxt#30399] [Questions]
nuxt version:  3.13.2,

_NOTE:_ I can't use useRuntimeCofig because I want the var value outside setup context

**Comment by cernymatej** [comment]
Could you share a little more about your use case? AFAIK, your only option is to use `process.env.SOME_VAR` outside Nuxt / Nitro context. This will be baked in at build-time, though.

Edit: I just noticed the date of the post 😆 I don't know why it popped up at the top of the list for me 👀

**Reply by Moataz-Abdulbaqi** [reply]
Thank you for the response. I was able to resolve the issue using `process.env`, as you suggested.

My apologies for the confusion — I’ll go ahead and close the discussion.
